monai/bundle/scripts.py

```diff
@@ -249,10 +413,13 @@ def verify_metadata(
     try:
         # the rest key-values in the _args are for `validate` API
         validate(instance=metadata, schema=schema, **_args)
-    except ValidationError as e:
+    except exceptions.ValidationError as e:
         # as the error message is very long, only extract the key information
         logger.info(re.compile(r".*Failed validating", re.S).findall(str(e))[0] + f" against schema `{url}`.")
         return
+    except exceptions.SchemaError as e:
```

Comment:
Does this change failed the CI tests?

Thanks.

Reply:
Hi Nic, without this change, `pylint` will raise error. https://github.com/Project-MONAI/MONAI/runs/6075933692?check_suite_focus=true

Reply:
I restored the code and tested with CI.

Thanks.